docker/Dockerfile.pdo-dev

```diff
@@ -47,18 +47,20 @@
 #   - for develooping based on source in host you might map source into container with an option
 #     like -v $(pwd):/project/pdo/src/private-data-objects/
 
-ARG UBUNTU_VERSION=bionic
+ARG UBUNTU_VERSION=18.04
 # 16.04 -> xenial, 17.10 -> artful, 18.04 -> bionic
 # NOTE: xenial might not work anymore (see below), preferred choice is bionic ..
 
 FROM ubuntu:${UBUNTU_VERSION}
 
-ARG UBUNTU_VERSION=bionic
+ARG UBUNTU_VERSION=18.04-server
+ARG UBUNTU_NAME=bionic
 # for bizare docker reason, we have to redefine it here ...
 
-ARG SGX_SDK=sgx_2.6
-ARG OPENSSL=1.1.0k
-ARG SGXSSL=v2.4.1
+ARG SGXSDK=2.9.1
+ARG SDKBIN=2.9.101.2
+ARG OPENSSL=1.1.1d
+ARG SGXSSL=2.9_1.1.1d
```

Comment:
Seems there is a newer lin_2.10_1.1.1g tag which goes with bug-fixed openssl version. Afaict, sdk 2.10 is not out yet but i guess imminent? Anyway, can change later ...

Reply:
Yeah, I think it makes sense to keep the SDK and SSL versions in sync.